scopes/workspace/workspace/workspace.ts

```diff
@@ -1212,6 +1210,13 @@ export class Workspace implements ComponentFactory {
     this.clearCache();
   }
 
+  getComponentPackagePath(component: ConsumerComponent | Component) {
```

Comment:
now that we have this function here, we can use it here (change I just merged)-
https://github.com/teambit/bit/blob/2d20e318013bfd5473084933938255943e684de9/scopes/preview/preview/preview.main.runtime.tsx#L139